scripts/2_create.sql

```diff
@@ -17,10 +17,11 @@ IF NOT EXISTS subnets
 );
 
 CREATE TABLE
-IF NOT EXISTS devices
+IF NOT EXISTS ips
 (
-    id INTEGER PRIMARY KEY,
-    ip INET,
+    id SERIAL PRIMARY KEY,
```

Comment:
Question: Is there any reason not to use the IP as the primary key?

Reply:
Nah not really.  I could have used the IP as primary key.  Doing it the way the PR has allows for duplicate IPs (as in, two or more "devices" have the same IP), which I suppose should never happen.  Do we trust our data source (IPAM) to never have such a condition?

I'm going to merge since this PR is approved.